CONTRIBUTING.md

```diff
@@ -124,17 +136,27 @@ Tycho has Maven dependencies to Equinox and JDT, so these artifact are used from
 
 ## 🏗️ Build & Test
 
-`mvn clean install -Pits`
+From the root directory of your local Tycho git-repository clone run the following Maven commands...
+* to check if compilation and all tests succeed:
+    * `mvn clean verify -Pits`
+* to install your version of Tycho into your local Maven repository (skips all tests for faster installation):
+    * `mvn clean install -DSkipTests`
+
+In order to test your changes of Tycho locally in a project-build, install your modified Tycho locally as described above
+and use the corresponding Tycho (probably snapshot) version in the project being build.
+You can also debug that build with the steps below (from here you can jump to step 3 immediately).
 
 ## Debugging 
 
 In order to debug Tycho plugins inside Eclipse:
 
 1. Get the Tycho sources in Eclipse
 2. Create/get a project that highlights the bug
-3. Run the project with `mvnDebug clean install`
```

Comment:
I think we should keep a note about mvnDebug here as alternative, or maybe fond a link to maven doc to reference that mentions such debug strategies.

Reply:
Agree on that. I added it as a second variant (IMO debugging the build directly from Eclipse is more convenient and therefore should be named first).

Unfortunately I did not found official doc about `mvnDebug` on the Apache Maven website.